vitals-monitor.test.mjs

```diff
@@ -2,8 +2,38 @@ import { expect } from 'chai';
 import { vitalsOk } from './vitals-monitor.mjs';
 
 describe('vitals checker', function () {
-  it('returns false when any vital is out of range ', async function () {
-    expect(await vitalsOk(99, 102, 70)).to.be.false;
-    expect(await vitalsOk(98.1, 70, 98)).to.be.true;
+  this.timeout(10000); // Increase timeout to 10 seconds

```

Comment:
Is this needed? Because you are setting the test mode to true everywhere - it will complete quickly, I guess